modules/custom/openy_upgrade_tool/config/install/views.view.openy_upgrade_dashboard.yml

```diff
@@ -0,0 +1,428 @@
+            more_link_text: ''
+            more_link_path: ''
+            strip_tags: false
+            trim: false
+            preserve_tags: ''
+            html: false
+          element_type: ''
+          element_class: ''
+          element_label_type: ''
+          element_label_class: ''
+          element_label_colon: true
+          element_wrapper_type: ''
+          element_wrapper_class: ''
+          element_default_classes: true
+          empty: ''
+          hide_empty: false
+          empty_zero: false
+          hide_alter_empty: true
+          destination: true
+          entity_type: logger_entity
+          plugin_id: entity_operations
+      filters:
+        type:
+          id: type
+          table: logger_entity
+          field: type
+          value:
+            openy_config_upgrade_logs: openy_config_upgrade_logs
+          entity_type: logger_entity
+          entity_field: type
+          plugin_id: bundle
+        field_config_path_value:
+          id: field_config_path_value
+          table: logger_entity__field_config_path
+          field: field_config_path_value
+          relationship: none
+          group_type: group
+          admin_label: ''
+          operator: 'not empty'
+          value: ''
+          group: 1
+          exposed: false
+          expose:
+            operator_id: ''
+            label: ''
+            description: ''
+            use_operator: false
+            operator: ''
+            identifier: ''
+            required: false
+            remember: false
+            multiple: false
+            remember_roles:
+              authenticated: authenticated
+          is_grouped: false
+          group_info:
+            label: ''
+            description: ''
+            identifier: ''
+            optional: true
+            widget: select
+            multiple: false
+            remember: false
+            default_group: All
+            default_group_multiple: {  }
+            group_items: {  }
+          plugin_id: string
+      sorts:
+        changed:
+          id: changed
+          table: logger_entity
+          field: changed
+          relationship: none
+          group_type: group
+          admin_label: ''
+          order: ASC
+          exposed: false
+          expose:
+            label: ''
+          granularity: second
+          entity_type: logger_entity
+          entity_field: changed
+          plugin_id: date
+      title: 'OpenY upgrade dashboard'
+      header:
+        area_text_custom:
+          id: area_text_custom
+          table: views
+          field: area_text_custom
+          relationship: none
+          group_type: group
+          admin_label: ''
+          empty: false
+          tokenize: false
+          content: "<h1>OpenY upgrade dashboard</h1>\n<p>These configs have been changed manually, that's why we cannot import them. Please fix this conflict.</p>\n<p>After conflict fixing please delete this config from list.</p>"
```

Comment:
```html
<h1>OpenY upgrade dashboard</h1>\n<p>These configs have been changed manually, that's why we cannot import them. Please fix conflicts below and then you'll get updates from OpenY for affected components.</p>\n<p>Upon fixing the conflict, please delete message from dashboard to let system know that conflict has been resolved, otherwise all updates for the component will be skipped.</p>
```